invenio.cfg

```diff
@@ -194,14 +194,26 @@ SECURITY_SEND_REGISTER_EMAIL = False
 # -------------------
 # See https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/config.py
 from invenio_oauthclient.contrib.keycloak import KeycloakSettingsHelper
-from cds_rdm.oidc import cern_info_serializer, confirm_registration_form
+from cds_rdm.oidc import (
+    cern_info_serializer,
+    confirm_registration_form,
+    cern_group_serializer,
+    cern_group_handler,
+)
+from urllib.parse import quote
+
+_base_url = os.environ.get("CERN_KEYCLOAK_BASE_URL", "https://keycloak-qa.cern.ch/")
 
 _keycloak_helper = KeycloakSettingsHelper(
     title="CERN",
     description="CERN SSO authentication",
-    base_url=os.environ.get("CERN_KEYCLOAK_BASE_URL", "https://keycloak-qa.cern.ch/"),
+    base_url=_base_url,
     realm="cern",
     app_key="CERN_APP_CREDENTIALS",
+    logout_url="{}auth/realms/cern/protocol/openid-connect/logout?redirect_uri={}".format(
+        _base_url,
+        quote(SITE_UI_URL)
```

Comment:
+1 on this, I think if you try to replace the value by env variable it wouldn't work because it would take the one from this file instead.

Reply:
One alternative is to use a "lazy"/LocalProxy to get the value from `current_app.config` like so:

```python
logout_url=LocalProxy(
    lambda: "{}auth/realms/cern/protocol/openid-connect/logout?redirect_uri={}".format(
        _base_url,
        quote(current_app.config["SITE_UI_URL"])
    )
```

Not sure if this will work though, depends on when `logout_url` is accessed (hopefully not outside application context).

Reply:
From review along with @anikachurilova +1 +1 to needs lazy loading

Reply:
Thanks for pointing this out! 
LocalProxy doesn't seem to work fine, I've used `os.environ`, unless someone has a better suggestion.
![Screenshot from 2023-06-19 18-58-54](https://github.com/CERNDocumentServer/cds-rdm/assets/15194802/c6e46a57-c91c-4469-a0b2-23202ae8de85)

Reply:
Oof, this was indeed really "lazy" all the way until before adding the redirect header 🤦 

Ok, last suggestion: could we make that the `logout_url` config is a string OR function/callable? In that case it's "lazy" up to the point of actually performing the redirect in the `/oauth/logout` endpoint, where you could call the function.

This is not blocking for merging at all, so we can open also just open an issue.